README.md

```diff
@@ -1 +1,32 @@
-# WeatherApp
\ No newline at end of file
+# WeatherApp
+
+## Our Team:
+
+- Abdalhakim Abumusameh
+- Amira Alnajjar
+- Abdallah Rabah
+
+## Live Link:
+## Figma Link:
+https://www.figma.com/file/DG5hjc4MhTM6NPr0OKZcp8/final-project?node-id=0%3A1
```

Comment:
Figma link should be  👇 👇 
[Figma](https://www.figma.com/file/DG5hjc4MhTM6NPr0OKZcp8/final-project?node-id=0%3A1)

[Figma](figma link)